src/components/MAPI/apps/ClusterDetailReleaseApps.tsx

```diff
@@ -0,0 +1,104 @@
+import { useAuthProvider } from 'Auth/MAPI/MapiAuthProvider';
+import { usePermissionsForReleases } from 'MAPI/releases/permissions/usePermissionsForReleases';
+import { supportsReleases } from 'MAPI/utils';
+import { GenericResponseError } from 'model/clients/GenericResponseError';
+import * as applicationv1alpha1 from 'model/services/mapi/applicationv1alpha1';
+import * as releasev1alpha1 from 'model/services/mapi/releasev1alpha1';
+import React, { useEffect, useMemo, useRef } from 'react';
+import { FlashMessageType } from 'styles';
+import useSWR from 'swr';
+import FlashMessageComponent from 'UI/Display/FlashMessage';
+import ErrorReporter from 'utils/errors/ErrorReporter';
+import { useHttpClientFactory } from 'utils/hooks/useHttpClientFactory';
+
+import ClusterDetailAppList from './ClusterDetailAppList';
+import { IAppsPermissions } from './permissions/types';
+import { mapDefaultApps } from './utils';
+
+interface IClusterDetailReleaseAppsProps {
+  appList?: applicationv1alpha1.IAppList;
+  isLoading: boolean;
+  isClusterCreating?: boolean;
+  errorMessage: string;
+  appsPermissions: IAppsPermissions;
+  releaseVersion: string;
+}
+
+const ClusterDetailReleaseApps: React.FC<
+  React.PropsWithChildren<IClusterDetailReleaseAppsProps>
+> = ({
+  appList,
+  isLoading,
+  isClusterCreating = false,
+  errorMessage,
+  appsPermissions,
+  releaseVersion,
+}) => {
+  const provider = window.config.info.general.provider;
+  const clientFactory = useHttpClientFactory();
+  const auth = useAuthProvider();
+
+  const releaseClient = useRef(clientFactory());
+  const { canGet: canGetReleases } = usePermissionsForReleases(
+    provider,
+    'default'
+  );
+
+  const isReleasesSupportedByProvider = supportsReleases(provider);
```

Comment:
Thanks for checking if releases are supported before fetching them! In the case that they aren't though, would it still make sense to render this component?

Reply:
Actually, this component will be used only on vintage. For CAPI providers there will be another component where we will fetch `default-apps-xxx` bundle. So this check is redundant probably.